sound/soc/sof/topology.c

```diff
@@ -146,11 +148,19 @@ static int sof_keyword_dapm_event(struct snd_soc_dapm_widget *w,
 	dev_dbg(sdev->dev, "received event %d for widget %s\n",
 		event, w->name);
 
+	/* get runtime PCM params using widget's stream name */
+	spcm = snd_sof_find_spcm_name(sdev, swidget->widget->sname);
+
 	/* process events */
 	switch (event) {
 	case SND_SOC_DAPM_PRE_PMU:
+		if (spcm->stream[stream].suspend_ignored) {
+			dev_dbg(sdev->dev, "KWD stream already active in S0ix, no need to resume\n");
```

Comment:
@keyonjie lets fix all comments to not say active. Lets use suspended or not suspended?

Reply:
It was from Kai, OK, changing it now.